### DISCUSSION TITLE
Elimine el codigo sin querer, pero lo habia commiteado antes

### DISCUSSION THREAD

**Discussion by Lautymendez** [INGCOM-UNRN-P1/cursada-2023#44] [Preguntas y respuestas] [answered]
Buenass, por una confusion elimine un codigo que ya tenia hecho pero por suerte, ya habia hecho un commit .
Quiero recuperarlo, ¿donde puedo ir a ese commit? 
Con el comando git log pude sacar el ID pero nose donde ir para revisar el codigo.

**Comment by martinvilu** [comment, marked as answer]
1. ¿esta en la recicladora de windows?
2. `git status` que dice?

Tenés dos opciones, 
* podes ir a github y bajar el archivo
* ejecutá el siguiente comando `git checkout HEAD <filename>`, reemplazando `<filename>` por el nombre del archivo (sin las `<>`)
 - Quedaría algo como  `git checkout HEAD ejercicioborrado.c`, probablemente

Esto 'descarga' de lo último que esté guardado del archivo que indicas. (entre comillas porque sale del repositorio en la computadora, no de internet)

**Comment by Vittatat** [comment]
thanks
